### PR TITLE
Issue #17: Glide Integration

### PR DESCRIPTION
**Pull request by SalosV**
**Integrate Glade:** #17  
- Add gradle dependencies as Glide 

**Testing Done:**

* On Emulator: Huawei P20 Lite.

**How To Test:**

* add an imageView in the layout activity_main
* in the file MainActivity.kt declare the view and pass it the image

Captura de Pantalla 2019-11-25 a la(s) 2 41 21 p  m

**Signed-off-by:**

`Santiago Lozada <santiagolozada95@gmail.com>`

### CODE REVIEW

File: presentation/src/main/java/com/architect/coders/mu8/MainActivity.kt [hunk 1]
```diff
@@ -8,5 +8,6 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
```
Comment (aliceresponde) [inline presentation/src/main/java/com/architect/coders/mu8/MainActivity.kt]: Remove  it.

File: presentation/src/main/java/com/architect/coders/mu8/Extensions.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.architect.coders.mu8
+
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+fun ImageView.loadUrl(url: String) {
```
Comment (aliceresponde) [inline presentation/src/main/java/com/architect/coders/mu8/Extensions.kt]: @SalosV Do you want to use a placeHolder for the image or how can we manage if glide has an error while the image is loading from URL? (callback) ? 
@recaldev

**Review by recaldev — COMMENTED** [review]
Code seems fine, will test the branch once I can before approval.

File: presentation/build.gradle [hunk 3]
```diff
@@ -35,6 +35,11 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
 
+    //Glide
```
Comment (aliceresponde) [inline presentation/build.gradle]: update gradle

File: presentation/build.gradle [hunk 4]
```diff
@@ -34,6 +34,16 @@ dependencies {
     implementation SupportDependencies.coreKtx
     implementation SupportDependencies.constraintLayout
 
+    implementation LibsDependencies.glide
```
Comment (aliceresponde) [inline presentation/build.gradle:37]: Sort the dependencies in block related to the same dependency object.

**Review by aliceresponde — APPROVED** [review]
LG

**Pull request merged**